### DISCUSSION TITLE
Installing magma on GEEKOM MINI PC

### DISCUSSION THREAD

**Discussion by africa243** [magma/magma#15440] [General]
Hello Guys
Magma is really new to me. Is it possible to deploy magma on GEEKOM Mini PC XT13 Pro, 13th Gen Intel i9-13900H Mini Computers(14 C/20T, Up to 5.4GHz) 32GB DDR4/2TB PCIe Gen 4 SSD Windows 11 Pro Support Wi-Fi 6E/Bluetooth 5.2/USB 4.0/2.5G LAN/8K/Quad Displays
? T
The idea is to run it commercially in rural LTE deployment with 1K subscribers. 
Also is there any existing hardware available on the market that comes with pre-installed Magma to handle 1k+ users?

**Comment by lucasgonze** [comment]
Let's have a Slack conversation. @mickymkumar has some of the background info you need.
